source/add-price-feed.md

```diff
@@ -0,0 +1,62 @@
+# Add a price feed
```

Comment:
I'll review it better in local, and maybe i'll create a commit on top of it. Thx Dani!